src/test/HostActivationTests/GivenThatICareAboutLightupAppActivation.cs

```diff
@@ -128,6 +128,82 @@ public void Muxer_activation_of_LightupApp_WithLightupLib_Succeeds()
                 .HaveStdOutContaining("Hello LightupClient");
         }
 
+        // Success case with roll-backwards support
+        [Fact]
+        public void Muxer_activation_of_LightupApp_WithLightupLib_and_Roll_Backwards_Succeeds()
```

Comment:
Test for prerelease to release rollback

Reply:
Test that we don't roll over minor.

Reply:
Sure. I will add that, however I'll need to refactor the "shared fx" test code (from `GivenThatICareAboutMultilevelSharedFxLookup.cs`) into a common class so I can deploy and target a specific preview\non-preview framework version (instead of the current one like the current test).